### DISCUSSION TITLE
"Lite" license

### DISCUSSION THREAD

**Discussion by ejnark** [bitgapp/eqMac#956] [Ideas]
I'd like to use this app solely to apply an oratory1990 EQ to a pair of headphones. I have no problems paying for that, but 40 euros is overkill for what I need. I'd pay 10-20 bucks for a "Lite" license without thought. I don't care about any of the other Pro features, I just want to apply an EQ preset.

Perhaps there are more people like me. Adding a "Lite" license might increase your sales reach.

**Comment by nodeful** [comment]
Hey there,

You can apply the oratory1990 and others EQ presets in Advanced EQ with no License at all.

Kind Regards,
Roman - Creator of eqMac
